src/main/java/features/TaskList.java

```diff
@@ -0,0 +1,32 @@
+package features;
+
+import java.util.ArrayList;
+
+import tasks.Task;
+
+/**
+ * Contains and modifies tasks entered by the user.
+ */
+public class TaskList {
+    protected ArrayList<Task> list;
```

Comment:
Plural form should be used for a collection of objects. 

E.g
`int[] values;`

Reply:
Same as above, i've made the change now.